### PR TITLE
Move babel injection code into polymer-build.

### PR DESCRIPTION
**Pull request by rictic**
- [x] CHANGELOG.md has been updated

Compare with https://github.com/Polymer/polymer-cli/pull/720

After this lands, we can remove that code from cli.

### CODE REVIEW

File: src/test/inject-babel-helpers_test.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+      entrypoint: 'index.html',
+      shell: 'shell.html',
+      sources: [
+        'source-dir/**',
+      ],
+    });
+  });
+
+  test('injects babel helpers inline into the entrypoint', (done) => {
+    const webcomponentsLoaderFilname = 'webcomponents-loader.js';
```
Comment (usergenic) [inline src/test/inject-babel-helpers_test.ts]: `webcomponentsLoaderFilname` -> `webcomponentsLoaderFileName`

File: src/streams.ts [hunk 2]
```diff
@@ -44,6 +44,29 @@ export function waitForAll(streams: NodeJS.ReadableStream[]):
 }
 
 /**
+ * Returns the string contents of a Vinyl File object, waiting for
+ * all chunks if the File is a stream.
+ */
+export async function getFileContents(file: File): Promise<string> {
+  if (file.isBuffer()) {
+    return file.contents.toString('utf-8');
+  } else if (file.isStream()) {
```
Comment (usergenic) [inline src/streams.ts]: Yikes I don't think we check for this anywhere else.  I added personal TODO to update all the file contents-based code to use `getFileContents`

File: src/inject-babel-helpers.ts [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+import {AsyncTransformStream, getFileContents} from './streams';
+
+import File = require('vinyl');
+
+const p = dom5.predicates;
+
+const scriptOrImport = p.OR(
+    p.hasTagName('script'),
+    p.AND(p.hasTagName('link'), p.hasSpaceSeparatedAttrValue('rel', 'import')));
```
Comment (usergenic) [inline src/inject-babel-helpers.ts]: I added personal TODO to update other html import selectors to use hasSpaceSeparatedAttrValue.  Thanks for doing that here.

Comment (rictic) [inline src/inject-babel-helpers.ts]: done!

File: src/test/inject-babel-helpers_test.ts (hunk 1 above)
Comment (rictic) [inline src/test/inject-babel-helpers_test.ts]: Done

**Pull request merged**